packages/p2p/src/components/online-status/online-status.scss

```diff
@@ -32,11 +32,15 @@
 
         & .online-status__icon {
             position: absolute;
-            bottom: -0.1px;
+            bottom: -0.3px;
             right: -0.1px;
             margin: 0;
             border-radius: 50%;
             transform: scale(1.1);
+
+            @include mobile {
+                bottom: -0.1px;
+            }
```

Comment:
Fix issue where the online status icon is misaligned in desktop:
![Screenshot 2022-11-29 at 3 07 59 PM](https://user-images.githubusercontent.com/103016120/204465692-e2486cc1-8fbc-4570-86ee-d40be43b92d0.png)